### PR TITLE
Misc cleanup and TODOs on malware checks.

### PR DESCRIPTION
**Pull request by xmunoz**
- Change backfill function to invoke `IMalwareCheckService` interface
- Add support for `kwargs` to `IMalwareCheckService` interface
- Rename variable from reserved word `file` to `release_file`
- Add `FatalCheckException` for non-retryable exceptions
- Replace `MALWARE_CHECK_BACKEND` in dev/environment

### CODE REVIEW

File: warehouse/malware/checks/utils.py [hunk 1]
```diff
@@ -78,3 +78,7 @@ def extract_file_content(archive_stream, file_path):
                 return None
         except tarfile.TarError:
             return None
+
+
+class FatalCheckException(Exception):
```
Comment (woodruffw) [inline warehouse/malware/checks/utils.py]: Thanks for adding this!

File: tests/unit/malware/test_services.py [hunk 2]
```diff
@@ -31,13 +32,14 @@ def test_create_service(self):
         service = PrinterMalwareCheckService.create_service(None, request)
         assert service.executor == print
 
-    def test_run_checks(self, capfd):
+    @pytest.mark.parametrize(("kwargs"), [{}, {"manually_triggered": True}])
+    def test_run_checks(self, capfd, kwargs):
```
Comment (ewdurbin) [inline tests/unit/malware/test_services.py]: oh neat, til `capfd` pytest fixture.

File: warehouse/malware/checks/utils.py (hunk 1 above)
Comment (ewdurbin) [inline warehouse/malware/checks/utils.py]: could this be moved to `warehouse/malware/checks/errors.py` or `warehouse/malware/errors.py`?

File: warehouse/malware/tasks.py [hunk 3]
```diff
@@ -47,6 +49,9 @@ def run_check(task, request, check_name, obj_id=None, manually_triggered=False):
 
     try:
         check.run(**kwargs)
+    except FatalCheckException as exc:
```
Comment (ewdurbin) [inline warehouse/malware/tasks.py]: great call on being able to explicitly error a check out!

**Pull request merged**